### PR TITLE
Change URDE to Metaforce

### PR DESCRIPTION
**Pull request by BoofOof32**
Project has renamed, just thought I’d update

### CODE REVIEW

File: games/m.yaml [hunk 1]
```diff
@@ -347,6 +347,26 @@
   updated: 2016-03-16
   url: https://www.meridiannext.com
 
+- name: Metaforce
+  lang:
+  - C++
+  - Python
+  - C
```
Comment (cxong) [inline games/m.yaml:354]:
```suggestion
```
the C code appears to be 3rd party headers only

File: games/m.yaml [hunk 2]
```diff
@@ -347,6 +347,26 @@
...
+  originals:
+  - Metroid Prime
+  repo: 'https://github.com/AxioDL/metaforce'
+  type: remake
+  url: 'https://axiodl.com/'
+  status: playable
+  info: engine only
+  updated: '2022-03-01'
+  images:
+  - https://github.com/AxioDL/metaforce/blob/main/assets/metaforce-screen1.png?raw=true
```
Comment (cxong) [inline games/m.yaml:367]:
```suggestion
  - https://github.com/AxioDL/metaforce/blob/main/assets/metaforce-screen1.png?raw=true
  - https://axiodl.com/images/hecl_hu25398d5c0d2da88c4a5221db09696c79_1724775_0x480_resize_bg292a2d_box_2.png
  - https://axiodl.com/images/pwe_hu7c7bec678fa5eef23b41aaa434dfa04c_1742668_0x480_resize_bg292a2d_box_2.png
  video:
    youtube: uMwqpL2_-LY
```

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/m.yaml (hunk 2 above)
Comment (BoofOof32) [reply games/m.yaml:367]: The 2 extra images are actually Prime World Editor, not necessarily Metaforce

Comment (BoofOof32) [reply games/m.yaml:367]: I will include them though since they are related projects

**Review by cxong — APPROVED** [review]

**Pull request merged**